fastestimator/pipeline.py

```diff
@@ -321,15 +321,19 @@ def get_results(self, mode: str = "train", epoch: int = 1, num_steps: int = 1,
             results = results[0]
         return results
 
-    def get_loader(self, mode: str, epoch: int = 1,
-                   shuffle: Optional[bool] = None) -> Union[DataLoader, tf.data.Dataset]:
+    def get_loader(self,
+                   mode: str,
+                   epoch: int = 1,
+                   shuffle: Optional[bool] = None,
+                   output_keys: Optional[Set[str]] = None) -> Union[DataLoader, tf.data.Dataset]:
         """Get a data loader from the Pipeline for a given `mode` and `epoch`.
 
         Args:
             mode: The execution mode for the loader. This can be 'train', 'eval' or 'test'.
             epoch: The epoch index for the loader. Note that epoch indices are 1-indexed.
             shuffle: Whether to shuffle the data. If None, the value for shuffle is based on mode. NOTE: This argument
                 is only used with FastEstimator Datasets.
+            output_keys: What keys can be produced from pipeline. If None, all keys will be considered.
```

Comment:
do we need to specify that this only works with our op dataset an not with tf data loader?

Reply:
I don't think it's needed, because the "key" concept only matters when using Ops.